x-pack/plugins/security_solution/server/lib/telemetry/endpoint_task.ts

```diff
@@ -241,14 +248,19 @@ export class TelemetryEndpointTask {
     const { body: failedPolicyResponses } = (endpointData.epPolicyResponse as unknown) as {
       body: EndpointPolicyResponseAggregation;
     };
-    const policyResponses = failedPolicyResponses.aggregations.policy_responses.buckets.reduce(
-      (cache, endpointAgentId) => {
-        const doc = endpointAgentId.latest_response.hits.hits[0];
-        cache.set(endpointAgentId.key, doc);
-        return cache;
-      },
-      new Map<string, EndpointPolicyResponseDocument>()
-    );
+
+    // If there is no policy responses in the 24h > now then we will continue
+    const policyResponses =
+      failedPolicyResponses.aggregations === undefined
```

Comment:
Always like to double check this can't be null as well?

Reply:
Thanks! Updated in https://github.com/elastic/kibana/pull/106269/commits/be52925c13bac2095e42c910f3fdd6311eb5c3c3